sdk/datashare/Microsoft.Azure.Management.DataShare/src/Properties/AssemblyInfo.cs

```diff
@@ -8,8 +8,8 @@
 [assembly: AssemblyTitle("Microsoft Azure Data Share Management Library")]
 [assembly: AssemblyDescription("Provides Microsft Azure DataShare management functions for managing the Microsoft Azure DataShare service.")]
 
-[assembly: AssemblyVersion("1.2.0.0")]
-[assembly: AssemblyFileVersion("1.2.0.0")]
+[assembly: AssemblyVersion("1.0.2.0")]
```

Comment:
Is the version downgrade intentional? Going from `1.2.0` to `1.0.2`

We only allow upgrading versions, except for special cases. Is this a special circumstance?

Reply:
I don't want to downgrade. I want to keep the 1.2.0 version I published last week. All I need to do is to refresh/update the 2019 version and release 1.0.2. The previous version of 2019 was 1.0.1.
Basically, I just want to create a 1.0.2 in the public nuget packages page here:
https://www.nuget.org/packages/Microsoft.Azure.Management.DataShare.

Will this PR overrwrite/delete the 1.2.0 version I published last week?

Reply:
@JonathanCrd  Please let me know if this PR doesn't affect or overwrite the 1.2.0 version nuget.Otherwise, I'll how can I release a 1.0.2 release for the 2019-11-01 package tag?

Thannks

Reply:
As @allenjzhang suggested, the typical github hotfix process would work here. Go to your last old release (via tags or commit id) and branch it into a hotfix branch. Make the modification there. You can run release pipeline against that branch. 

This is to avoid confusion. Instead of using the version 1.2.0 as a base, use version 1.0.1

Reply:
@JonathanCrd Can you provide the github hotfix process steps please?